qemu/tests/virtio_fs_group_permission_access.py

```diff
@@ -51,8 +51,13 @@ def run(test, params, env):
 
     try:
         for _username in username:
-            cmd = add_user_cmd % _username
-            process.run(cmd)
+            if process.system("id %s" % _username, shell=True,
+                              ignore_status=True) == 0:
+                output = process.run(del_user_cmd % _username)
```

Comment:
hi @xiagao , how about using 's, o = process.getstatusoutput(del_user_cmd % _username, shell=True)', then if s != 0, check if error msg contains the msg 'is currently used by process', if it has, then we give an error(test.error), else, we give a failure(e.g. test.fail('Unknown error when deleting the user: %s' % o)

The problem here is we skip the  unknown error( expect 'used by process') .

Reply:
Thank you, updated.